engine/runner/src/main/java/org/enso/runner/Utils.java

```diff
@@ -210,7 +210,7 @@ private static <S> void printFrame(
   static String adjustCwdToProject(String fileToRun) {
     assert fileToRun != null;
     if (!ImageInfo.inImageRuntimeCode()) {
-      return null;
+      return System.getProperty("enso.user.dir");
```

Comment:
- the 17e2200d6c change is unrelated to `_JAVA_OPTIONS` changed
- but while testing various combinations of launcher I found out that
- with _native image_ `enso` launcher the following execution fails:
- `sbt:enso> runEngineDistribution --jvm --run test/Base_Tests`
- saying it cannot find `test/Base_Tests` project
- the problem is that the NI part 
   - properly resolves the project
   - changes CWD to the `test` directory
   - invokes the JVM `main`
- however the CWD for the JVM part is already `test` directory
- and resolving `test/Base_Tests` in that directory doesn't work
- this change fixes the problem by providing the JVM part the location of the original CWD via a property